src/virtualenv/create/via_global_ref/_virtualenv.py

```diff
@@ -55,13 +56,24 @@ def find_spec(self, fullname, path, target=None):
                     try:
                         spec = find_spec(fullname, path)
                         if spec is not None:
-                            old = spec.loader.exec_module
+                            # https://www.python.org/dev/peps/pep-0451/#how-loading-will-work
+                            spec.loader = deepcopy(spec.loader)  # loaders may be shared, create new that also patches
```

Comment:
this just moves the bug slightly:
- not all loaders are copyable
- some loaders depend on being singletons and copying them breaks that invariant